terraform/rds.tf

```diff
@@ -1,15 +1,16 @@
 resource "aws_db_instance" "postgres" {
-  allocated_storage      = var.db_storage
-  engine                 = "postgres"
-  engine_version         = "12.5"
-  db_subnet_group_name   = aws_db_subnet_group.db.id
-  vpc_security_group_ids = [aws_security_group.db.id]
-  deletion_protection    = var.db_delete_protection
-  name                   = var.db_name
-  username               = var.db_username
-  password               = var.db_password
-  instance_class         = var.db_instance_class
-  storage_encrypted      = var.db_storage_encrypted
-
+  identifier                      = "${var.env}-beacons-database"
+  allocated_storage               = var.db_storage
+  engine                          = "postgres"
+  engine_version                  = "12.5"
+  db_subnet_group_name            = aws_db_subnet_group.db.id
+  vpc_security_group_ids          = [aws_security_group.db.id]
+  deletion_protection             = var.db_delete_protection
+  name                            = var.db_name
+  username                        = var.db_username
+  password                        = var.db_password
+  instance_class                  = var.db_instance_class
+  storage_encrypted               = var.db_storage_encrypted
+  skip_final_snapshot             = var.db_skip_final_snapshot
```

Comment:
We had an issue when trying to destroy a stack, where we couldn't delete the RDS instance because it needed a `final snapshot`, since we didn't have this option specified before.

It was a bit of a pain! But we eventually managed to apply this change to the RDS then destroy it with some Googling and [this](https://ndench.github.io/terraform/terraform-destroy-rds)